### ISSUE TITLE
TWDT set captain

### ISSUE THREAD

**Issue by connermcd** [Trench-Wars/twcore#108]
## Trac Ticket [#1216](http://twcore.org/ticket/1216) new enhancement

> Reported by: board
> Owned by:  
> Component:   Bots - TWD/TWL
> Priority:    lowest
> Severity:    trivial
## Description:

TL;DR version at bottom.   something i've noticed and I think is a relative easy and fast patch, the old ways of using basebot and wbduelbut etc, was to !t1-setcaptain name,  but, since I've been staff, it was taught to me, and many others, that we !setcaptain 0:name and !setcaptain 1:name.  What I'm getting at, is that sometimes during TWDT matches, if it's a new host, they are trying to use !setcaptain instead of the matchbot's !t1-setcaptain command, and become confused, normally it'd be no issue for a person to dig and discover how to properly set caps, BUT with TWDT, the game has been !started and the timer to add players has began, all while the host is stressing out to figure out how to set a cap.  2 options 1st one being please allow !setcaptain to work alongside of !t1-setcaptain, as it does in ?go base, or option 2, edit the !help menu to display the command fully and properly, not !setcaptain use -t1 prefix   or w/e.  simpler = better! 

TL;DR add !setcaptain 0:name and !setcaptain 1:name to matchbot for TWDT purposes (allow !t1-setcaptain commands to still work along side it)

**Comment by connermcd** [comment]
_board changed **priority** from_ normal _to_ lowest

**Comment by connermcd** [comment]
**board>** please note: this is the same matchbot used in TWD matches, not the twdt specific matchbot, those are not used.
